packages/fluent/scss/drawer/index.scss

```diff
@@ -0,0 +1,26 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "drawer",
+    dependencies: (
+        "overlay",
+        "icon"
```

Comment:
please check if we need to add the toolbar and list as deps

Reply:
No other dependencies need to be added.